runtime/moonbase/tests/integration_test.rs

```diff
@@ -407,15 +407,15 @@ fn reward_block_authors() {
 		.build()
 		.execute_with(|| {
 			set_parachain_inherent_data();
-			for x in 2..599 {
+			for x in 2..1199 {
```

Comment:
I'd prefer to express these in terms of the configuration values like `5 * LeaveDelegatorsDelay` or whatever value this is. But that can be for a followup.

Reply:
In my `deferred staking` PR (#1035 ) I added a `roll_to_round_begin` which takes a round number rather than a block number. That's only loosely related to this...